Configuration is not visible to KafkaEventListenerProviderFactory

Hello

I'm using Keycloak in Kubernetes, using this [Helm chart](https://artifacthub.io/packages/helm/bitnami/keycloak)

I'm using `add-kafka-config.cli` (adjustting config file name) - the configs are added.
Also, I've added environment variables to the pod.

When finally I put a jar into `deployments` - I receive an error

```
[org.jboss.msc.service.fail] (MSC service thread 1-2) MSC000001: Failed to start service jboss.deployment.unit."keycloak-kafka-1.1.0-jar-with-dependencies.jar".POST_MODULE: org.jboss.msc.service.StartException in service jboss.deployment.unit."keycloak-kafka-1.1.0-jar-with-dependencies.jar".POST_MODULE: WFLYSRV0153: Failed to process phase POST_MODULE of deployment "keycloak-kafka-1.1.0-jar-with-dependencies.jar"
        at org.jboss.as.server@15.0.1.Final//org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:189)
        at org.jboss.msc@1.4.12.Final//org.jboss.msc.service.ServiceControllerImpl$StartTask.startService(ServiceControllerImpl.java:1739)
        at org.jboss.msc@1.4.12.Final//org.jboss.msc.service.ServiceControllerImpl$StartTask.execute(ServiceControllerImpl.java:1701)
        at org.jboss.msc@1.4.12.Final//org.jboss.msc.service.ServiceControllerImpl$ControllerTask.run(ServiceControllerImpl.java:1559)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.ContextClassLoaderSavingRunnable.run(ContextClassLoaderSavingRunnable.java:35)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor.safeRun(EnhancedQueueExecutor.java:1990)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.doRunTask(EnhancedQueueExecutor.java:1486)
        at org.jboss.threads@2.4.0.Final//org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1363)
        at java.base/java.lang.Thread.run(Thread.java:829)
Caused by: java.lang.NullPointerException: topic must not be null.
        at deployment.keycloak-kafka-1.1.0-jar-with-dependencies.jar//com.github.snuk87.keycloak.kafka.KafkaEventListenerProviderFactory.init(KafkaEventListenerProviderFactory.java:56)
        at org.keycloak.keycloak-services@14.0.0//org.keycloak.services.DefaultKeycloakSessionFactory.loadFactories(DefaultKeycloakSessionFactory.java:297)
        at org.keycloak.keycloak-services@14.0.0//org.keycloak.services.DefaultKeycloakSessionFactory.deploy(DefaultKeycloakSessionFactory.java:154)
        at org.keycloak.keycloak-services@14.0.0//org.keycloak.provider.ProviderManagerRegistry.deploy(ProviderManagerRegistry.java:42)
        at org.keycloak.keycloak-wildfly-server-subsystem@14.0.0//org.keycloak.subsystem.server.extension.KeycloakProviderDeploymentProcessor.deploy(KeycloakProviderDeploymentProcessor.java:58)
        at org.jboss.as.server@15.0.1.Final//org.jboss.as.server.deployment.DeploymentUnitPhaseService.start(DeploymentUnitPhaseService.java:182)
        ... 8 more
```

Basically `Scope config` is empty in `KafkaEventListenerProviderFactory.init(..)`

I've pathced the plugin to read configs parameters directly from environment variables via `System.getenv(..)` - and this version works just fine.

Could you please help on the issue ? I can use patched version, but I'd prefer to help debugging and fixing it.

--- a/README.md
+++ b/README.md
@@ -17,7 +17,7 @@ Simple module for [Keycloak](https://www.keycloak.org/) to produce keycloak even
 
 Kafka version: `2.12-2.1.x`, `2.12-2.4.x`, `2.12-2.5.x`, `2.13-2.8`
 
-Keycloak version: `4.8.3`, `6.0.x`, `7.0.0`, `9.0.x`, `10.0.x`, `13.0.x`, `14.0.x`
+Keycloak version: `4.8.3`, `6.0.x`, `7.0.0`, `9.0.x`, `10.0.x`, `13.0.x`, `14.0.x` `15.0.x`
 
 Java version: `11`, `13`
 

--- a/pom.xml
+++ b/pom.xml
@@ -4,7 +4,7 @@
 	<modelVersion>4.0.0</modelVersion>
 	<groupId>com.github.snuk87.keycloak</groupId>
 	<artifactId>keycloak-kafka</artifactId>
-	<version>1.1.0</version>
+	<version>1.1.1</version>
 
 	<properties>
 		<maven.compiler.source>11</maven.compiler.source>

--- a/pom.xml
+++ b/pom.xml
@@ -9,7 +9,7 @@
 	<properties>
 		<maven.compiler.source>11</maven.compiler.source>
 		<maven.compiler.target>11</maven.compiler.target>
-		<keycloak.version>14.0.0</keycloak.version>
+		<keycloak.version>15.0.2</keycloak.version>
 		<kafka.version>2.8.0</kafka.version>
 	</properties>
 

--- a/src/main/java/com/github/snuk87/keycloak/kafka/KafkaEventListenerProviderFactory.java
+++ b/src/main/java/com/github/snuk87/keycloak/kafka/KafkaEventListenerProviderFactory.java
@@ -41,12 +41,12 @@ public class KafkaEventListenerProviderFactory implements EventListenerProviderF
 	@Override
 	public void init(Scope config) {
 		LOG.info("Init kafka module ...");
-		topicEvents = config.get("topicEvents");
-		clientId = config.get("clientId", "keycloak");
-		bootstrapServers = config.get("bootstrapServers");
-		topicAdminEvents = config.get("topicAdminEvents");
+		topicEvents = config.get("topicEvents", System.getenv("KAFKA_TOPIC"));
+		clientId = config.get("clientId", System.getenv("KAFKA_CLIENT_ID"));
+		bootstrapServers = config.get("bootstrapServers", System.getenv("KAFKA_BOOTSTRAP_SERVERS"));
+		topicAdminEvents = config.get("topicAdminEvents", System.getenv("KAFKA_ADMIN_TOPIC"));
 
-		String eventsString = config.get("events");
+		String eventsString = config.get("events", System.getenv("KAFKA_EVENTS"));
 
 		if (eventsString != null) {
 			events = eventsString.split(",");

